aksel.nav.no/website/components/website-modules/preview/LiveQueryProvider.tsx

```diff
@@ -0,0 +1,31 @@
+import { LiveQueryProvider as SanityLiveQueryProvider } from "next-sanity/preview";
+import { useMemo } from "react";
+import { getClient } from "@/sanity/client.server";
+import LiveQuery from "./LiveQuery";
+import { PreviewProps } from "./Preview.types";
+
+const LiveQueryProvider = ({
+  query,
+  params,
+  props,
+  children,
+}: PreviewProps) => {
+  const client = useMemo(() => {
+    return getClient().withConfig({
+      useCdn: false,
+      ignoreBrowserTokenWarning: true,
+      perspective: "previewDrafts",
+      withCredentials: true,
+    });
+  }, []);
```

Comment:
Can this be moved outside the component?

Reply:
Since the file is lazy loaded, can you call `previewDraftsClient()` outside the component, so that you don't have to use `useMemo`?

```ts
const client = previewDraftsClient();

const LiveQueryProvider = ({
  query,
  params,
  props,
  children,
}: PreviewProps) => {
  return (
    <SanityLiveQueryProvider client={client}>
      (...)
```

Reply:
Done and tested 🚀